aws/ec2metadata/api_ops.go

```diff
@@ -160,3 +160,11 @@ type EC2InstanceIdentityDocument struct {
 	RamdiskID          string    `json:"ramdiskId"`
 	Architecture       string    `json:"architecture"`
 }
+
+func suffixPath(base, add string) string {
```

Comment:
Is this issue specific to this service and may not arise for others. If not, should we consider moving this to another package as an utility?